sites/public/cypress/mockData/applicationData.ts

```diff
@@ -307,6 +277,107 @@ export const ElmVillageApplication: Application = {
   language: LanguagesEnum.en,
 }
 
+const BlueSkyProgramsAndPreferences = {
+  preferences: [],
+  programs: [
+    {
+      claimed: false,
+      key: "servedInMilitary",
+      options: [
+        {
+          key: "servedInMilitary",
+          checked: false,
+        },
+        {
+          key: "doNotConsider",
+          checked: false,
+        },
+        {
+          key: "preferNotToSay",
+          checked: false,
+        },
+      ],
+    },
+    {
+      key: "tay",
+      claimed: false,
+      options: [
+        {
+          key: "tay",
+          checked: false,
+        },
+        {
+          key: "doNotConsider",
+          checked: false,
+        },
+        {
+          key: "preferNotToSay",
+          checked: false,
+        },
+      ],
+    },
+    {
+      key: "disabilityOrMentalIllness",
+      claimed: false,
+      options: [
+        {
+          key: "disabilityOrMentalIllness",
+          checked: false,
+        },
+        {
+          key: "doNotConsider",
+          checked: false,
+        },
+        {
+          key: "preferNotToSay",
+          checked: false,
+        },
+      ],
+    },
+    {
+      key: "housingSituation",
+      claimed: false,
+      options: [
+        {
+          key: "notPermanent",
+          checked: false,
+        },
+        {
+          key: "homeless",
+          checked: false,
+        },
+        {
+          key: "doNotConsider",
+          checked: false,
+        },
+        {
+          key: "preferNotToSay",
+          checked: false,
+        },
+      ],
+    },
+    {
+      key: "rentBasedOnIncome",
+      claimed: false,
+      options: [
+        {
+          key: "flatRent",
+          checked: false,
+        },
+        {
+          key: "30Percent",
+          checked: false,
+        },
+      ],
+    },
+  ] as ApplicationMultiselectQuestion[],
+}
+
+export const autofillBlueSkyApplication: Application = {
+  ...ElmVillageApplication,
+  ...BlueSkyProgramsAndPreferences,
+}
+
 export const minimalDataApplication: Application = {
```

Comment:
chore: Looks like the minimalDataApplication is no longer used. Could we delete this?